website/content/en/preview/concepts/deprovisioning.md

```diff
@@ -7,36 +7,34 @@ description: >
 ---
 
 Karpenter sets a Kubernetes [finalizer](https://kubernetes.io/docs/concepts/overview/working-with-objects/finalizers/) on each node it provisions.
-The finalizer specifies additional actions the Karpenter controller will take in response to a node deletion request.
-These include:
-
-* Marking the node as unschedulable, so no further pods can be scheduled there.
-* Evicting all pods other than daemonsets from the node.
-* Terminating the instance from the cloud provider.
-* Deleting the node from the Kubernetes cluster.
+The finalizer signals to the Karpenter Termination controller to gracefully shutdown a node that receives a deletion request. Karpenter deprovisioning can be executed by Karpenter through the Deprovisioning Controler or by the user through manual deprovisioning.
```

Comment:
```suggestion
The finalizer blocks deletion of the node object while the termination controller cordons and drains the node, before removing the underlying machine. Deprovisioning is triggered by deprovisioning controller or by a user 
 or external system that sends a DELETE request to the node object.
```

Reply:
Took some bits and added it in. Let me know what you think.